.travis.yml

```diff
@@ -42,4 +42,4 @@ before_install:
 
 install: true
 
-script: travis_wait 30 mvn clean install -DskipTests -T 4 -q -Pitests
+script: mvn clean install -DskipTests -q -Pitests
```

Comment:
Not related to this task.  Please remove.

Reply:
@belugabehr Retaining travis_wait 30 will cause the check to fail due to compilation timeout. Do you have any good suggestions?